src/elements/content-sidebar/stories/tests/MetadataSidebarRedesign-visual.stories.tsx

```diff
@@ -41,7 +42,7 @@ export const AddTemplateDropdownMenuOn = {
     play: async ({ canvasElement }) => {
         const canvas = within(canvasElement);
 
-        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 5000 });
+        const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' }, { timeout: 2000 });
```

Comment:
so i was experimenting with this on your PR and I noticed this works, I think we should avoid using timeout to a hardcoded ms if possible
```
        waitFor(async () => {
            const addTemplateButton = await canvas.findByRole('button', { name: 'Add template' });
            expect(addTemplateButton).toBeInTheDocument();
            await userEvent.click(addTemplateButton);

            const customMetadataOption = await canvas.findByRole('option', { name: 'Virus Scan' });
            expect(customMetadataOption).toBeInTheDocument();
            await userEvent.click(customMetadataOption);

            const templateHeader = await canvas.findByRole('heading', { name: 'Virus Scan' });
            expect(templateHeader).toBeInTheDocument();
        });
```

Reply:
Hmmm, when I was running it like so I was running into timeouts.
lately they became so long that no story was working on Chrome for me. I've noticed that we have a call to global metadata templates unmocked. I've added it with empty array and it made all the stories much faster to load, almost instant.

I think the reason it worked for you and not us (in PL) was that unmocked calls were slower due to round trips.
Now it should be fixed.